server/portal/apps/workspace/api/views.py

```diff
@@ -247,21 +247,31 @@ def post(self, request, *args, **kwargs):
 
             if settings.DEBUG:
                 wh_base_url = settings.WH_BASE_URL + '/webhooks/'
-                # jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
+                jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
             else:
                 wh_base_url = request.build_absolute_uri('/webhooks/')
-                # jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
+                jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
 
             job_post['parameterSet']['envVariables'] = job_post['parameterSet'].get('envVariables', []) + [{'key': '_webhook_base_url', 'value':  wh_base_url}]
 
             portal_name = settings.PORTAL_NAMESPACE
             job_post['tags'] = job_post.get('tags', []) + [portal_name]
 
-            # TODOv3 Webhooks/notifications continues
-            # job_post['notifications'] = [
-            #     {'url': jobs_wh_url,
-            #      'event': e}
-            #     for e in settings.PORTAL_JOB_NOTIFICATION_STATES]
+            # TODOv3: change ttlMinutes to be dynamic based on max job runtime
+            job_post["subscriptions"] = [
+               {
+                    "description": "Email notification",
+                    "enabled": False,
```

Comment:
Don't we want `enabled` to be `True`?

Reply:
Yeah it makes sense for this to be True. But the subscription still works with this being false. I'll confirm with the tapis team whether or not this is a bug